<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapSetting</class>
 <widget class="QDialog" name="MapSetting">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Setting</string>
  </property>
  <widget class="QDialogButtonBox" name="mapSettingButtonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="mf_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="mf_name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="settingsForm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>39</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="outputTabForm">
    <attribute name="title">
     <string>Output</string>
    </attribute>
    <widget class="QComboBox" name="mf_units">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>53</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="units_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Units</string>
     </property>
    </widget>
    <widget class="QLabel" name="size_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="mf_sizewidth_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>34</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>width</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="mf_sizewidth">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="mf_sizeheight_label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>32</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>height</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="mf_sizeheight">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>31</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QComboBox" name="mf_default_outputformat">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>7</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="mf_default_output_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Default output format</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="debugTabForm">
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set-up the debug level.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>Debug</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="advancedTabForm">
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Advanced settings for map object (configuration files, specific parameters).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mapSettingButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapSettingButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
